app/javascript/gobierto_visualizations/webapp/containers/subsidies/SubsidiesIndex.vue

```diff
@@ -47,9 +47,9 @@ export default {
     updateFilteredItems(value) {
       this.value = value
       if (!this.value) {
-        this.items = this.subsidiesData.filter(contract => contract.beneficiary_name.toLowerCase().includes(this.value.toLowerCase()))
+        this.items = this.subsidiesData.filter(contract => contract.beneficiary.toLowerCase().includes(this.value.toLowerCase()))
```

Comment:
Always that you apply functions/mutations to something you've to provide fallback, in this case destructuring.

```suggestion
        this.items = this.subsidiesData.filter(({ beneficiary = "" }) => beneficiary.toLowerCase().includes(this.value.toLowerCase()))
```

Reply:
Done!